src/Entity/Formation/Article.php

```diff
@@ -133,4 +148,40 @@ public function setAxe(Axe $axe): void
     {
         $this->axe = $axe;
     }
+
+    /**
+     * @return Collection|File[]
+     */
+    public function getFiles(): Collection
+    {
+        return $this->files;
+    }
+
+    public function setFiles(array $files): self
+    {
+        if (\count($files) > 0) {
+            foreach ($files as $file) {
+                $this->addFile($file);
+            }
+        }
+
+        return $this;
```

Comment:
We do not use fluent pattern on this project.

Please remove all of these `return $this;` from your setters. ( setters return types should be `: void` )